deploy/kube/localregistry.yaml

```diff
@@ -0,0 +1,88 @@
+# after creating the local registry, to push images to it you need to open up a port to the pod's registry:
```

Comment:
We definitely can rely on docker hub/gcr - the mixer.yaml in this PR uses gcr. I was using this for my own testing so I didn't have to push random images to dockerhub with my own code changes. I think it's worth keeping around for local debugging but I can remove it if we don't agree.